modules/ztp-ztp-custom-resources.adoc

```diff
@@ -9,64 +9,60 @@ Zero touch provisioning (ZTP) uses custom resource objects to extend the Kuberne
 
 A custom resource definition (CRD) file defines your own object kinds. Deploying a CRD into the managed cluster causes the Kubernetes API server to begin serving the specified custom resource for the entire lifecycle.
 
-For each custom resource in the `<site>.yaml` file on the managed cluster, the data is used to create installation custom resources in a directory named
-for the cluster.
+For each custom resource in the `<site>.yaml` file on the managed cluster, the data is used to create installation custom resources in a directory named for the cluster.
 
-On the cluster site, an automated Discovery image ISO file creates a directory with the site name and a file with the cluster name. The cluster file contains the custom resources shown in the following table. Every cluster has its own namespace, and all of the custom resources are under that namespace. The namespace and the custom resource names match the cluster name.
+ZTP provides two ways for defining and installing custom resources on managed clusters: a manual approach when you are provisioning a single cluster and an automated approach when provisioning multiple clusters.
```

Comment:
Would it be appropriate to bullet list these?

"ZTP provides two ways for defining and installing custom resources on managed clusters: 

* A manual approach when you are provisioning a single cluster.
*  An automated approach when provisioning multiple clusters."

Reply:
It may if they weren't immediately followed by description lists of the terms.